src/Cli.Tests/EndToEndTests.cs

```diff
@@ -755,4 +755,30 @@ public void TestBaseRouteIsConfigurableForSWA(string authProvider, bool isExcept
             Assert.AreEqual("/base-route", runtimeConfig.Runtime.BaseRoute);
         }
     }
+
+    /// <summary>
+    /// Test to validate that whenever the flag rest.request-body-not-strict is included in the init command,
+    /// the runtimeconfig is initialized such that the runtime allows extraneous fields in the request body for REST.
+    /// </summary>
+    /// <param name="isRequestBodyNotStrict">Boolean variable indicating whether or not to include the
```

Comment:
depending on whether any changes are made with regards to my CLI feedback, this param name may change because having it in the negative is more confusing that just saying `isRequestBodyStrict` == true/false